### ISSUE TITLE
Correctly resolve wildcards in subpath imports

### ISSUE THREAD

**Issue by JavaScriptBach** [lukeed/resolve.exports#34]
See https://github.com/jestjs/jest/issues/15250

**Comment by SimenB** [comment]
Specifically https://github.com/jestjs/jest/issues/15250#issuecomment-2396097163

```js
const { imports } = require('resolve.exports');

const resolved = imports({
  'imports': {
    '#test/*.css': './src/*.css',
    '#test/*.ts': './src/*.ts',
  },
}, '#test/asdf/css.ts');

console.log(resolved);
```

Logs out `[ './src/asdf.css' ]` instead of `['./src/asdf/css.ts']`

**Comment by lukeed** [comment]
Closed by #37 & available in `2.0.3`

**Comment by JavaScriptBach** [comment]
Excellent, thank you!
